lib/Doctrine/KeyValueStore/Storage/RiakStorage.php

```diff
@@ -68,62 +69,75 @@ public function requiresCompositePrimaryKeys()
         return false;
     }
 
+    private function store($storageName, $key, array $data)
+    {
+        $namespace = new RiakNamespace('default', $storageName);
```

Comment:
`$namespace` and `$location` creation can be put into a private method too. It is used in 3 different methods.